src/pages/docs/certificate-manager/acme-enterprise/when-acme.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: When to Use ACME in Enterprise Networks
+html_title: When to Use ACME in Enterprise Networks
+description: When can the ACME protocol be used to issue and renew certificates in enterprise networks.
+---
+
+At Smallstep we love the ACME protocol.
+If we could, we would advise to always use it to issue certificates.
+Unfortunately, not every certificate management use case can be implemented using the ACME protocol.
+The following sections describe the prerequisite requirements and some scenarios in which the ACME protocol can be used to issue certificates.
+We also describe some use cases for which ACME is less or not suitable, accompanying pointers to solutions for those use cases.
+
+### At a High Level
+
+As described in ["why you should use ACME"](/docs/certificate-manager/acme-enterprise/why-acme), the ACME protocol requires ACME clients to request certificates via HTTPS.
+They must also be able to respond to challenges from an ACME server.
+Requesting a certificate and responding to challenges involves multiple requests and responses, each of which can fail.
+A certificate cannot be issued if one of the requests or responses fails to be sent or received in time.
+
+For a certificate to be issued for an identifier, the ACME server contacts the system the identifier points to, either via DNS, HTTP, or HTTPS, to verify that the ACME client controls the requested identifier.
+If a client cannot respond to at least one of the challenges for the identifier, the ACME server won't issue a certificate.
+In case multiple identifiers are requested in a single certificate, at least one challenge for every identifier must be successful, for the certificate to be issued.
+
+The ACME specification currently only officially supports domain names and IP addresses in certificate requests and challenges.
+Certificate requests for domain names and IP addresses are currently supported in the Smallstep platform.
+
+
+### Scenarios
+
+The below listing is an overview of exemplary scenarios in which the ACME protocol can be used to issue certificates:
```

Comment:
I also think of fully airgapped networks for medical systems, industrial control systems, etc. I'm not sure how common these are though.

Reply:
For the other scenarios I had the hosted platform in mind, but for these it would be the on-prem solution, correct?

Reply:
Yeah they would need to use on-prem